### DISCUSSION TITLE
[Question] MQTT easist way to get notified when recording started

### DISCUSSION THREAD

**Discussion by jokakilla** [blakeblackshear/frigate#11615] [Ask A Question] [answered]
### What is your question:

Hi,
I hope this question hasn't been asked yet. At least I didn't find anything in the discussions.

Basically I want to get notified on my smartphone when a recording has started.

Currently I subscribe to the frigate/events endpoint via MQTT and filter for events of type new. Every "new" event triggers a notification with a snapshot. But there sometimes are a lot of events e.g. a person with a dog is entering the image. At least two events (dog and person) are triggered. This way I get notified reliably but get lot's of unnecessary notifications too.

In the example where dog and person enter the image the recording starts as soon as the dog enters and stops after both have left the image. Only want to get one notification in such scenario.

I thought about putting a rate limiter per camera on the notification code but I guess there must be a better way.
Any tips?

**Comment by NickM-27** [comment, marked as answer]
this type of thing is addressed in the next version of frigate (0.14) with the new /reviews mqtt topic

**Comment by jokakilla** [comment]
Oh man you're so quick. Thanks a lot. Exactly what I was searching for :)
